pkg/controller/podgroup.go

```diff
@@ -38,29 +38,29 @@ import (
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog/v2"
 
-	pgv1 "sigs.k8s.io/scheduler-plugins/pkg/apis/podgroup/v1alpha1"
-	pgclientset "sigs.k8s.io/scheduler-plugins/pkg/generated/clientset/versioned"
-	pginformer "sigs.k8s.io/scheduler-plugins/pkg/generated/informers/externalversions/podgroup/v1alpha1"
-	pglister "sigs.k8s.io/scheduler-plugins/pkg/generated/listers/podgroup/v1alpha1"
+	schedulingv1alpha1 "sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
```

Comment:
nit: the aliases can be shortened to "schedXYZ" (save us 5 characters :))

Reply:
Done